### PR TITLE
SSR all asides without hydrating them

### LINKED ISSUE #902: Asides have FOUC
Asides are no longer SSRd and now cause FOUC

### PR DESCRIPTION
**Pull request by e111077**
Fixes #902

Testable links:

https://pr897-8869287---lit-dev-5ftespv5na-uc.a.run.app/tutorials/intro-to-lit/#2
https://pr897-8869287---lit-dev-5ftespv5na-uc.a.run.app/docs/getting-started/#use-bundles

### CODE REVIEW

File: packages/lit-dev-content/src/components/litdev-tutorial.ts [hunk 1]
```diff
@@ -426,7 +429,19 @@ export class LitDevTutorial extends LitElement {
           html`${`Could not fetch step description. Invalid URL /${
             this._projectLocation
           }/#${this._idxToSlug(this._idx)}`}`,
-        complete: (response) => html`${unsafeHTML(response)}`,
+        complete: (response) => {
+          // Must use DOM parser to attach declarative shadow roots
+          const domResponse = (domParser.parseFromString as any)(
```
Comment (aomarks) [inline packages/lit-dev-content/src/components/litdev-tutorial.ts:434]: why is the `as any` needed? Something wrong with the `parseFromString` signature?

File: packages/lit-dev-content/src/components/litdev-tutorial.ts [hunk 2]
```diff
@@ -426,7 +429,19 @@ export class LitDevTutorial extends LitElement {
...
+            response,
+            'text/html',
+            {includeShadowRoots: true}
+          );
+          const body = domResponse.body;
+          if (needsDSDPolyfill) {
+            hydrateShadowRoots(body);
+          }
+          return Array.from(body.children);
```
Comment (aomarks) [inline packages/lit-dev-content/src/components/litdev-tutorial.ts:443]: Nit:

```suggestion
          return [...body.children];
```

**Review by aomarks — CHANGES_REQUESTED** [review]
Awesome!

**Review by AndrewJakubowicz — APPROVED** [review]
Manual tests across Safari/Firefox and Chrome look great!

File: packages/lit-dev-content/src/components/litdev-tutorial.ts (hunk 1 above)
Comment (e111077) [reply packages/lit-dev-content/src/components/litdev-tutorial.ts:434]: the options argument is not in TS yet. I assume because it's only Chrome for now

Comment (aomarks) [reply packages/lit-dev-content/src/components/litdev-tutorial.ts:434]: Ah ok. Could you add a comment?

Comment (e111077) [reply packages/lit-dev-content/src/components/litdev-tutorial.ts:434]: aye aye

**Review by aomarks — APPROVED** [review]
Great!

**Pull request merged**